app/helpers/dashboard_helper.rb

```diff
@@ -18,7 +18,7 @@ def dashboard_actions
           widgets_to_add
         end
       ),
-      documentation_button,
+      documentation_button('', :class => 'pull-right'),
       auto_refresh_button(:defaults_to => true)
```

Comment:
I think the real problem is here - `auto_refresh_button` creates layout that is different then the documentation button, and requires all sorts of overrides. If you change it so the auto-refresh button is just an `a` without the surrounding `div` it should play nicely with the others and no need for the special margin.
Then add `.btn-toolbar .btn` to the selector [here](https://github.com/theforeman/foreman/blob/develop/app/assets/stylesheets/patternfly_and_overrides.scss#L146) so none of the buttons will be floated, and then they will show in the order defined in this helper (right now some are floated and some aren't).